text/0000-rfc-issue-generation.md

```diff
@@ -0,0 +1,158 @@
+In addition, it usually requires the person merging the RFC to hunt down (or ping) any maintainers for sub-teams that haven't created relevant issues.
+
+The goal of this RFC is to minimize the overhead of creating issues for maintainers and the individual merging the RFCs by incoorporating the processes into the review process as opposed to the merge process.
+
+# What it is
+[what-it-is]: #what-it-is
+
+The proposal would be to add the following information to the end of the RFC template:
+
+```markdown
+# Issues to Create
+
+<!-- THIS SECTION IS INTENDED TO BE FILLED BY MAINTAINERS DURING THE RFC PROCESS. -->
+
+Issues listed below will be created once this RFC is approved and merged:
+
+<!--
+  FORMAT: - <repo>: <title> [labels]...
+  EXAMPLES:
+      - buildpacks/lifecycle: Implement something
+      - buildpacks/pack: Support this thing [good first issue][status/triage]
+-->
+```
+
+# How it Works
+[how-it-works]: #how-it-works
+
+<!--
+This is the technical portion of the RFC, where you explain the design in sufficient detail.
+The section should return to the examples given in the previous section, and explain more fully how the detailed proposal makes those examples work.
+-->
+
+During the review process, as maintainers review the RFC they can [suggest changes](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) to this section by adding a line in the following format:
+
+```
+- <repo>: <title> [labels]...
+```
+
+Once, the RFC is approved and ready to merge the [`merge-rfc.sh`](https://github.com/buildpacks/rfcs/blob/main/merge-rfc.sh) can parse the list in this section and create the respective issues.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+<!--
+Why should we *not* do this?
+-->
+
+- The RFC author is required to accept the suggested changes, preferably with proper DCO signatures. This would be adding a bit of toil on the authors behalf.
+
+# Alternatives
+[alternatives]: #alternatives
+
+
+### Use comments instead
+
+Instead of maintainers creating "suggested edits" to the RFC which then require the author to merge with proper DCO signing, 
+the maintainers could simply comment on the RFC. Once the RFC is merged the [`merge-rfc.sh`](https://github.com/buildpacks/rfcs/blob/main/merge-rfc.sh) script could detect issues by looking through the comments.
+
+```text
+FORMAT:
+  - /add-issue <repo>: <title> [labels]...
+  - /remove-issue <permalink-to-comment>
+```
+
+Examples:
+
+```text
+/add-issue buildpacks/pack: implement this feature [good first issue]
+```
+
+```text
+/remove-issue https://github.com/... 👈 link to comment
+```
+
+##### Pros
+
+- No need for maintainers to suggest edits and ensure author adds issues.
+
+##### Cons
+
+- No call-to-action for maintainers. The maintainers would have to remember to do this during their review.
```

Comment:
Perhaps the RFC merge script could fail if it detects a total of 0 net issues created.

Reply:
I'm not sure this would be a healthy check to impose. Looking through some RFCs, in particular those related to process or governance changes, it's unclear that any issues should be created. If we think hard enough, if only to satisfy the requirement, I'm sure we can find some issue to create but not sure it's the best approach.

Some examples:

- https://github.com/buildpacks/rfcs/blob/main/text/0071-cnb-user-research-2021.md
- https://github.com/buildpacks/rfcs/blob/main/text/0062-distribution-team.md
- https://github.com/buildpacks/rfcs/blob/main/text/0060-create-repo-issues.md
- https://github.com/buildpacks/rfcs/blob/main/text/0059-label-rfcs.md